SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -290,6 +291,54 @@ bool ValidateIdentifierNames(TArray<TSharedPtr<FUnrealType>>& TypeInfos)
 	return bSuccess;
 }
 
+bool ValidateAlwaysWriteRPCs(TArray<TSharedPtr<FUnrealType>>& TypeInfos)
```

Comment:
```suggestion
bool ValidateAlwaysWriteRPCs(const TArray<TSharedPtr<FUnrealType>>& TypeInfos)
```

Reply:
Don't we have a typedef for this container type?

Reply:
Looks like we don't! Though there's only 4 usages of this, all in this file, so I don't think there's a need to typedef.